### PR TITLE
Add cumulative and expected partition distribution plots

### PR DESCRIPTION
**Pull request by jpsmith5**

### CODE REVIEW

File: R/partition-plots.R [hunk 1]
```diff
@@ -99,25 +153,394 @@ calcPartitions = function(query, partitionList, remainder="intergenic") {
...
+		message("\tfound ", sum(ol>0))
+		partition[partition==0][ol > 0] = partitionNames[pi]
+	}
+    # Remove remainder
+	partition = partition[!partition=="0"]
+    tpartition = table(partition)
+	expectedPartitions = data.table::data.table(name=factor(names(tpartition)),
+                                                observed=as.vector(tpartition))
+    expectedPartitions[,expected:=partitionCounts$N]
+    expectedPartitions[,log10OE:=log10(expectedPartitions$observed/
```
Comment (nsheff) [inline R/partition-plots.R:229]: the indentation looks a bit off in this section.

**Comment by nsheff** [comment]
@jpsmith5 this is much more like what I was expecting, looks great. Can you add demonstrations of each of these plots to the vignette?

**Comment by jpsmith5** [comment]
Yes, have worked on the vignette and will add that. Meant to do that before opening this request.

**Comment by jpsmith5** [comment]
Alright. Vignette should be set. All plots can handle single GRanges as query or GRangesList now.  Should behave very similarly to the base calc/plotPartitions now.

**Comment by nsheff** [comment]
vignette looks great thanks

**Pull request merged**